components/marketing_signup_modal/index.coffee

```diff
@@ -21,11 +21,12 @@ class MarketingSignupModalInner extends Backbone.View
     @$('.marketing-signup-modal-error').hide()
     @$('form button').addClass 'is-loading'
     $.ajax
-      url: "#{sd.API_URL}/api/v1/user"
+      url: sd.AP.signupPagePath
       method: 'POST'
       data:
         email: @$('[name=email]').val()
         password: @$('[name=password]').val()
+        acquisition_initiative: sd.MARKETING_SIGNUP_MODAL_SLUG
```

Comment:
Here we pass through marketing campaign which gets [picked up in Artsy Passport](https://github.com/artsy/artsy-passport/pull/86/files#diff-280ede0624544d5719f1e4e31b6488e7R18) for tracking to segment upon signup.